L'exécution de votre programme a déclenché une exception java.lang.ArrayIndexOutOfBoundsException : 3
 Merci de corriger votre code.
   at plm.runtime18.Turmite.step (plm.runtime18.Turmite:45)
   at plm.runtime18.Turmite$$anonfun$run$2.apply$mcVI$sp (plm.runtime18.Turmite:62)
   at scala.collection.immutable.Range.foreach$mVc$sp (Range.scala:141)
   at plm.runtime18.Turmite.run (plm.runtime18.Turmite:60)
   at plm.core.lang.JVMCompiledLang.runEntity (null:-1)
   at plm.universe.World$1.run (null:-1)
   at java.lang.Thread.run (null:-1)
